src/arith_uint256.cpp

```diff
@@ -179,6 +180,17 @@ std::string base_uint<BITS>::ToString() const
     return (GetHex());
 }
 
+template <unsigned int BITS>
+std::string base_uint<BITS>::ToBitString() const
+{
+    std::string bits = "";
+    for (int i = 0; i < WIDTH; i++)
+    {
+        bits = bits + std::bitset<WIDTH>(pn[i]).to_string();
```

Comment:
how about bits.append(std::bitset<WIDTH>(pn[i]).to_string()) to make this a little more efficient?  I wish we could hand bitset a bigger array and avoid this whole for loop.

Reply:
see above